docs/resources/ec2_transit_gateway_route_table_association.md

```diff
@@ -10,6 +9,60 @@ description: |-
 
 Resource Type definition for AWS::EC2::TransitGatewayRouteTableAssociation
 
+## Example Usage
+
+### First example
+Associates the specified attachment with the specified transit gateway route table. You can associate one route table with an attachment.
+```terraform
+#Creates a transit gateway route table association
+resource "awscc_ec2_transit_gateway_route_table_association" "transit_gateway_rt_association_example" {
+  transit_gateway_attachment_id  = awscc_ec2_transit_gateway_attachment.transit_gateway_attachment_example.id
+  transit_gateway_route_table_id = awscc_ec2_transit_gateway_route_table.transit_gateway_rt_example.id
+}
+
+#Create a transit gateway attachment
+resource "awscc_ec2_transit_gateway_attachment" "transit_gateway_attachment_example" {
+  subnet_ids         = [aws_subnet.example.id]
+  transit_gateway_id = awscc_ec2_transit_gateway.transit_gateway_example.id
+  vpc_id             = aws_vpc.example.id
+}
+
+# Create an VPC
+resource "aws_vpc" "example" {
```

Comment:
Thank you , @hlazrv . 
- Can we use the `awscc_ec2_vpc` and `awscc_ec2_subnet` in place of the AWS provider resource here ?
- (nit): You can shorten the resource names to `example` unless there are multiple resources in the same configuration.

Reply:
Updated